packages/yourloops/test/unit/lib/patient/hook.test.tsx

```diff
@@ -91,11 +91,25 @@ describe('Patient hook', () => {
 
   describe('filterPatients', () => {
     const pendingPatientTeam = createPatientTeam('pendingTeamId', UserInvitationStatus.pending)
-    const pendingPatient = createPatient('pendingPatient', [pendingPatientTeam], {} as Monitoring)
-    const basicPatient = createPatient('basicPatient1', [basicTeam])
-    const bigBrainPatient = createPatient('big brain', [basicTeam])
-    bigBrainPatient.profile.firstName = 'big brain'
-    const allPatients = [pendingPatient, basicPatient, bigBrainPatient]
+    const pendingPatient = createPatient('pendingPatient', [pendingPatientTeam], undefined, { birthdate: new Date(2001, 10, 19, 0, 0, 0, 0) })
```

Comment:
Useless hours, minutes, seconds and milliseconds for the whole test
```suggestion
    const pendingPatient = createPatient('pendingPatient', [pendingPatientTeam], undefined, { birthdate: new Date(2001, 10, 19) })
```